build-scripts/cassandra-dtest-pytest.sh

```diff
@@ -21,6 +21,7 @@ export CASSANDRA_DIR=${WORKSPACE}
 #Have Cassandra skip all fsyncs to improve test performance and reliability
 export CASSANDRA_SKIP_SYNC=true
 export TMPDIR="./tmp"
+export RUN_STATIC_UPGRADE_MATRIX=true
```

Comment:
wonder if it is better/safer to move this line to inside just the case for "dtest-upgrade" (i.e. lines 76-77) ?

Reply:
looks like the move is necessary, after running https://ci-cassandra.apache.org/view/patches/job/Cassandra-devbranch-dtest/146/console

Reply:
done